ElectronicObserver/Data/EquipmentDataMaster.cs

```diff
@@ -341,7 +341,9 @@ public bool IsAntiSubmarineAircraft
 	/// <summary> 水上電探かどうか </summary>
 	public bool IsSurfaceRadar => IsRadar && LOS >= 5;
 
+	public bool IsHighAccuracyRadar => IsRadar && Accuracy >= 8;
 
+	
```

Comment:
Useless empty line.

Reply:
there was already an empty space 
thought it was intended
Now removed